website/docs/about/support.md

```diff
@@ -15,14 +15,18 @@ We’re happy to provide support to HashiCorp product teams that use Helios. Use
 
 Join the [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) Slack channel to ask questions, request and provide feedback, or start a conversation with our team. As a globally-distributed team, there may be a slight delay in answering some questions, but we’re committed to responding in a timely manner.
 
-## Office hours
+## File an issue
```

Comment:
Open to feedback on the idea of whether or not we should combine all 3 of these into one or if having some granularity between bugs, features, icons makes sense even though they all go to the same place.

Reply:
I think we can combine them so long as it's clear that the one link/form will cover **all** their needs/use cases. Also not opposed to keeping them separate, but I realize that's more content for the consumer to scan.

Reply:
Ok. I combined them into 1. Some refinement might still make sense but I like that as a general direction.